mockup/patterns/relateditems/pattern.js

```diff
@@ -99,27 +99,30 @@ define([
       homeText: _t('home'),
       folderTypes: ['Folder'],
       selectableTypes: null, // null means everything is selectable, otherwise a list of strings to match types that are selectable
-      attributes: ['UID', 'Title', 'portal_type', 'path', 'getIcon'],
+      attributes: ['UID', 'Title', 'portal_type', 'path','getURL', 'getIcon','contenttype_class','is_folderish','review_state'],
```

Comment:
I dont think this is that slippery (what i can solve with low effort serverside ..), and btw: Iam used to steep slopes in Austria and in Software ;-)
But waiting for your suggestions ...